src/views/scans/__tests__/showScansModal.test.tsx

```diff
@@ -72,39 +79,43 @@ describe('Modal', () => {
     expect(screen.getByText('Loading scans')).toBeInTheDocument();
     expect(baseElement).toMatchSnapshot();
   });
+
   test('Modal 0 jobs populates properly', async () => {
     const { baseElement } = render(<ScansModal {...propsWithZeroScanJobs} />);
     expect(screen.getByText('0 scans have run')).toBeInTheDocument();
     expect(baseElement).toMatchSnapshot();
   });
+
   test('Modal 1 job populates properly', async () => {
     const { baseElement } = render(<ScansModal {...propsWithOneScanJob} />);
     expect(screen.getByText('1 scan has run')).toBeInTheDocument();
     expect(baseElement).toMatchSnapshot();
   });
+
   test('Modal multiple jobs populates properly', async () => {
     const { baseElement } = render(<ScansModal {...propsWithScanJobs} />);
     expect(screen.getByText('2 scans have run')).toBeInTheDocument();
     expect(baseElement).toMatchSnapshot();
   });
+
   test('Modal closes with escape', async () => {
     const user = userEvent.setup();
     render(<ScansModal {...props} />);
-    await user.type(
-      screen.getByText(`view.label-{"context":"scans-ids","name":${props.scan.id}}`),
-      `{${KeyTypes.Escape}}`
-    );
+    await user.type(screen.getByText(`view.label-{"context":"scans-ids","name":${props.scan.id}}`), '{Escape}');
     expect(props.onClose).toHaveBeenCalled();
   });
+
   test('Modal shows the close button', () => {
     render(<ScansModal {...props} />);
     expect(screen.getByRole('button', { name: 'Close' })).toBeInTheDocument();
   });
+
   test('Modal clicking close button calls onClose', async () => {
     render(<ScansModal {...props} />);
     await userEvent.click(screen.getByRole('button', { name: 'Close' }));
     expect(props.onClose).toHaveBeenCalled();
   });
+
   test('Modal clicking download button calls onDownload with proper report id', async () => {
     render(<ScansModal {...propsWithScanJobs} />);
     await userEvent.click(screen.getAllByText('Download')[0]);
```

Comment:
I modified this test because it was failing, but I completely agree this shouldn't be handled here. I will rollback the changes and we will address it at another opportunity.